### ISSUE TITLE
함수구현 

### ISSUE THREAD

**Issue by soobni** [soobni/OSS_team#2]
이제 함수구현이나 하다가 문제가 생기면 여기에서 얘기해요 =)

**Comment by soobni** [comment]
저희 color를 불러오는 파일을 하나 더 만들어서 그 파일도 따로 저장하고 불러오는 함수 만들어야 할것같아요~! 민주 학우님께서 제품 저장하시는 함수하시는데 색도 저장하는 함수 만들어 주실 수 있을 까요?

**Comment by soobni** [comment]
앗 아니다 저장하는 함수는 안만들어도 될것같아요! 그냥 color.txt 만들고 그걸 불러오는 것만 하면 될것 같아요 =) 제가 만들게욤!ㅎㅅㅎ

**Comment by soobni** [comment]
저희 구조체에 파일에 있는 색을 저장할 변수 하나를 더 만들어야 할 것같아요~! color파일을 읽어서 저장할 변수가 없어요. fcolor라고 변수 더 추가했습니다! 그리고 색을 선택하는 selectcolorNo()도 만들어야 할것같아서 만들었습니당!

**Comment by Park-Minjoo** [comment]
네 확인했습니다~ 말씀하신거 반영해서 다른 함수도 지금 구현할게요:)

**Comment by Park-Minjoo** [comment]
void readProduct(Product *p); //하나의 제품을 출력하는 함수

제가 처음에 WIKI 를 만들때 이 함수를 넣었었는데, 지금 와서 생각해보니 굳이 필요할까 생각이 드네요! 
어떻게 생각하세요?

**Comment by soobni** [comment]
음.. 상관은 없지만 이미 만들었으니깐 그냥 두어도 되지 않을까요?  =)

**Comment by Park-Minjoo** [comment]
네 일단은 냅두고 필요할 때 구현해요 ㅎㅎ

**Comment by soobni** [comment]
저 지금 프로젝트 실행 가능하도록 함수들은 고쳤는데 민주학우님께서 PR보내주신것을 merge를 하지 않았습니다. 혹시 다시 PR 보내주실수 있을까요? ㅠㅠ

**Comment by Park-Minjoo** [comment]
manager.c에 void eventPoduct(Product *p,int count){

	for(int i = 0; i < count; i++){
                if(p[i].weight == -1) continue;
		if(p[i].event ==1){
                	printf("%2d ", i+1);
                	readProduct(p[i]);
		}
        }
         printf("\n");

}
이 부분은 오타일까요? :)

**Comment by Park-Minjoo** [comment]
함수 바꾸신 걸 WIKI에도 업데이트 해주실 수 있나요?

**Comment by Park-Minjoo** [comment]
지금 제가 돌려본 결과로는
1. 수정할때 제품명은? 하고 \n을 받아버림
2. 이름검색 아직 X
3. 파일 불러올 때 맨 마지막줄 입력 X
입니다. 
제가 맡은 함수인 1,2번을 수정할 테니 loadData함수 부분인 3번 수정 가능할까요?

**Comment by soobni** [comment]
저기 어떤 부분이 오타일까요??
그리고 wiki에 업데이트 하겠습니다 =)

**Comment by soobni** [comment]
3번 수정 할겠습니다~!

**Comment by Park-Minjoo** [comment]
poduct인데 제가 예전에 고쳤습니다~

**Comment by Park-Minjoo** [comment]
1,2번 마무리 하고 eventProduct를 구현하는 main함수 만들어서 구현까지 마쳤습니다~

**Comment by soobni** [comment]
오늘 색 로딩하는거 수정했습니다! 저희 웬만한건 거의 다 된 것 같은데 color로 제품찾기와 personalcolor로 제품찾기가 조금 어색한것같아요! 이제 다듬기만 하면 될것같습니다~!

**Comment by soobni** [comment]
색로딩하는거에서 안된 이유가 create할 때 넘어가는 값이 slist[curcount++]이라서 색을 curcount부터 읽은 거더라구요 ㅎ

**Comment by soobni** [comment]
저희 color랑 personalcolor검색하는 것을 제품을 전체 다 보여주는게 아니라 고객이 색의 index를 입력하면 그 제품을 보여주는 형식으로 하는게 어떨까요?

**Comment by soobni** [comment]
지금 거의 완성했는데 확인해보시겠어요 ?? =)

**Comment by Park-Minjoo** [comment]
네~ 확인했는데 product.txt 파일이 없는 경우에 제대로 추가가 되지 않고 "데이터가 없습니다!"라고 떠서 그 부분 main.c 수정했습니다.

**Comment by Park-Minjoo** [comment]
그리고 파일이 존재하는 경우에 컴파일 잘 되지 않아서 그 부분 수정 진행하겠습니다!

**Comment by Park-Minjoo** [comment]
color랑 personal color보여주는 거 엄청 깔끔 하게 잘 바뀐 것 같아요!! 보기에도 편하고 검색할때도 좋은 것 같습니다~~:):)

**Comment by soobni** [comment]
네~ 감사합니다~~! 수정한거 확인했습니당!
